client-linuxapp/src/serviceinfo.rs

```diff
@@ -323,12 +324,15 @@ impl CommandInProgress {
         let mut cmd = Command::new(self.command.as_ref().unwrap());
         cmd.args(&self.args);
 
-        if !self.return_stdout {
+        /*Below code ignores stderr incase of failure, commenting this part*/
+        /*
+         if !self.return_stdout {
```

Comment:
If this is part of the bug you can safely delete it instead of commenting it

Reply:
done now ,thanks @7flying